application/console/controllers/CronController.php

```diff
@@ -27,18 +26,30 @@ public function actionRemoveOldUnverifiedRecords()
      *  - active users with a verified totp Mfa
      *  - active users with a verified u2f Mfa
      *
+     * If you need to debug the Google Analytics call, do this ...
+     *     $response = $analytics->setProtocolVersion('1')
+     *                           ->setDebug(true)
+     *     ...
+     *     ...
+     *     \Yii::warning([
+     *            'results' => $response->getDebugResponse(),
+     *     ]);
+     *
+     *
      * @throws \Exception
      */
     public function actionGoogleAnalytics()
     {
-        $trackingId = Env::get('GA_TRACKING_ID', null); // 'UA-12345678-12'
+        $trackingId = \Yii::$app->params('gaTrackingId'); // 'UA-12345678-12'
```

Comment:
That's interesting... I didn't know `params` worked as a function call (`params('something')`). I generally treat it as an array (`params['something']`).